.github/workflows/actions.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           native-image: true
       - name: Build
         env:
-          ACCESS_KEY: ${{ secrets.TEAMSCALE_ACCESS_KEY }}
+          TEAMSCALE_ACCESS_KEY: ${{ secrets.TEAMSCALE_ACCESS_KEY }}
```

Comment:
As we now support passing the access key via environment variable directly, we can also use this for the tests. Previously, we only read the access key from the environment variable to put it into the command line option for the tests. I adjusted the tests accordingly and also made the naming of the enviromnent variables consistent here for the access key which is read from the Github secret.